minio/minio
# gateway object tagging

## Description
Object Tagging for Gateway.
PR depends on [minio/minio-go#1294](https://github.com/minio/minio-go/pull/1290/) for PutObject.

## Motivation and Context
Tags for objects using minio gateway. 

## How to test this PR?
Run minio in gateway mode. Check results for commands get-object-tagging, put-object-tagging, delete-object-tagging, put-object with tagging, copy-object with tagging & create-multipart-upload with tagging (upload-part/complete-multipart-upload to complete the transfer).

## Types of changes
- [√ ] New feature (non-breaking change which adds functionality)

## Checklist:
- [√ ] Unit tests needed
